extras/shell/src/main/java/org/apache/rya/shell/util/RyaDetailsFormatter.java

```diff
@@ -72,30 +72,28 @@ public String format(StorageType storageType, final RyaDetails details) {
         report.append("  Temporal Index:\n");
         report.append("    Enabled: ").append( details.getTemporalIndexDetails().isEnabled() ).append("\n");
 
-        if(storageType == StorageType.ACCUMULO) {
-            report.append("  PCJ Index:\n");
-            final PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
-            report.append("    Enabled: ").append( pcjDetails.isEnabled() ).append("\n");
-            if(pcjDetails.isEnabled()) {
-                if(pcjDetails.getFluoDetails().isPresent()) {
-                    final String fluoAppName = pcjDetails.getFluoDetails().get().getUpdateAppName();
-                    report.append("    Fluo App Name: ").append(fluoAppName).append("\n");
-                }
+        report.append("  PCJ Index:\n");
```

Comment:
The Statistics section does not have anything to do with PCJs so it shouldn't be printed conditionally when PCJ is turned on. They are also an Accumulo only feature.

Reply:
that include Join Selectivity?

Reply:
Yea, everything under it.